### DISCUSSION TITLE
Container Sandboxes do not update / seem broken

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [codesandbox/codesandbox-client#5525] [General]
I made some file changes to https://codesandbox.io/s/hardcore-drake-hz7kw?file=/app/templates/application.hbs
which was forked from: https://codesandbox.io/s/github/NullVoxPopuli/ember-new-output/tree/stable-sans-linting

![image](https://user-images.githubusercontent.com/199018/109079058-5e27ba80-76cc-11eb-9cff-bfc485bc7e82.png)

The server booted, and then never updated the preview. There have been no errors in the console/terminal.
I tried opening a new terminal to check `git status` to see if my changes were even saved, but the terminal is just blank no prompt at all.
Also, the `Problems` tab (and the editor squiggly) show an eslint error.
 - my project doesn't have ESLint?
 - I fixed the error by finishing typing my import.
 
![image](https://user-images.githubusercontent.com/199018/109079264-b52d8f80-76cc-11eb-9301-1e92eac9fc07.png)

-----

Update:
 it looks like the files did save, but the running server seems to be forever cached somehow? even though I've restarted the container and the server?
 
 -----
 
 Update again:
 
 after clicking "Restart Server" again, it seems to work.
 
 What can be done about this? I've had absolutely the worst experience with container sandboxes :cry:

**Comment by andrasbacsai** [comment]
I've check your example [sandbox](https://codesandbox.io/s/hardcore-drake-hz7kw?file=/app/templates/application.hbs). 

Hot-reload is not working, unfortunately, but I see the new content if I reload the preview window.
